OpenTabletDriver.Configurations/Configurations/Huion/H951P.json

```
@@ -0,0 +1,39 @@
+{
+  "Name": "Huion H951P",
+  "Specifications": {
+    "Digitizer": {
+      "Width": 221.0,
+      "Height": 141.0,
+      "MaxX": 44197.0,
+      "MaxY": 27599.0
+    },
+    "Pen": {
+      "MaxPressure": 8191,
+      "Buttons": {
+        "ButtonCount": 2
+      }
+    },
+    "AuxiliaryButtons": {
+      "ButtonCount": 11
+    },
+    "Touch": null
+  },
+  "DigitizerIdentifiers": [
+    {
+      "VendorID": 9580,
+      "ProductID": 103,
+      "InputReportLength": 12,
+      "OutputReportLength": null,
+      "ReportParser": "OpenTabletDriver.Configurations.Parsers.UCLogic.UCLogicTiltReportParser",
```

Given the missing features reported in TABLETS.md, it is likely that another parser is more accurate. Please test the following change:
```suggestion
      "ReportParser": "OpenTabletDriver.Configurations.Parsers.UCLogic.UCLogicV2ReportParser",
```

Of course, TABLETS.md needs to be updated accordingly.